encoding/thrift/thriftrw-plugin-yarpc/fx.go

```diff
@@ -33,9 +33,24 @@ const fxTemplate = `
 <$pkgname := printf "%sfx" (lower .Name)>
 package <$pkgname>
 
-<$yarpc  := import "go.uber.org/yarpc">
+<$yarpc := import "go.uber.org/yarpc">
 <$thrift := import "go.uber.org/yarpc/encoding/thrift">
 <$client := import .ClientPackagePath>
+<$fx := import "go.uber.org/fx">
+
+// Params defines the dependencies for the <.Name> client.
+type Params struct {
+	<$fx>.In
+
+	Provider <$yarpc>.ClientConfig
+}
+
+// Result defines the output of this Fx module.
```

Comment:
It'd be nice to write a little more here:

- We can use `.Name` to say, "Result is the output...which includes a client for Foo.`
- Might also be nice to explain why it's a struct instead of just a single return value. If someone's looking at this, it's a small teachable moment.